### PR TITLE
Add tiled rosdep

### PR DESCRIPTION
**Pull request by Vuwij**
Please add the following dependency to the rosdep database.

## Package name:

tiled

## Package Upstream Source:

https://github.com/mapeditor/tiled

## Purpose of using this:

Used as a tile editor

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/bookworm/tiled
- Ubuntu: https://packages.ubuntu.com/
  - https://packages.ubuntu.com/focal/tiled
- Fedora: https://packages.fedoraproject.org/
  - https://packages.fedoraproject.org/pkgs/tiled/tiled/

ROSDISTRO NAME

# The source is here:

http://sourcecode_url

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: rosdep/base.yaml [hunk 1]
```diff
@@ -7961,6 +7961,10 @@ texmaker:
   gentoo: [app-office/texmaker]
   nixos: [texmaker]
   ubuntu: [texmaker]
+tiled:
```
Comment (emersonknapp) [inline rosdep/base.yaml:7964]: Would you mind adding the arch/alpine/opensuse rules as suggested by the tooling? Otherwise this looks fine

**Review by emersonknapp — DISMISSED** [review]

File: rosdep/base.yaml [hunk 2]
```diff
@@ -7961,6 +7961,10 @@ texmaker:
...
+  debian: [tiled]
+  fedora: [tiled]
+  ubuntu: [tiled]
```
Comment (kjeremy) [inline rosdep/base.yaml:7967]: nixos also provides `tiled`:  https://search.nixos.org/packages?channel=23.05&show=tiled&from=0&size=50&sort=relevance&type=packages&query=tiled

**Comment by quarkytale** [comment]
@Vuwij friendly ping, are you still planning to take this forward?

File: rosdep/base.yaml (hunk 2 above)
Comment (nuclearsandwich) [inline rosdep/base.yaml:7967]:
```suggestion
tiled:
  alpine: [tiled]
  archlinux: [tiled]
  debian: [tiled]
  fedora: [tiled]
  nixos: [tiled]
  opensuse: [tiled]
  ubuntu: [tiled]
```

**Review by nuclearsandwich — APPROVED** [review]
Added the suggestions from CI and review @Vuwij if this is still valuable to you please consider accepting the suggestion to move the PR forward. Otherwise we will close it next week. Thanks.

**Review by clalancette — APPROVED** [review]

**Pull request merged**